src/checkpoint/traits/serializable_traits.h

```diff
@@ -171,9 +176,12 @@ struct SerializableTraits {
   static constexpr auto const has_serialize_function =
     has_serialize_instrusive or has_serialize_noninstrusive;
 
+  static constexpr auto const is_constructible =
+    is_default_constructible or is_reconstructible or is_tagged_constructible;
```

Comment:
This is missing the `is_nonintrusive_reconstructible` case I think

Reply:
Good point, I should rename `is_reconstructible` as `is_intrusive_reconstructible` to make that clear and create the OR-combo as `is_reconstructible`